daffodil-core/src/main/scala/org/apache/daffodil/runtime1/SequenceTermRuntime1Mixin.scala

```diff
@@ -76,7 +77,8 @@ trait ChoiceBranchImpliedSequenceRuntime1Mixin { self: ChoiceBranchImpliedSequen
       None,
       Maybe.Nope,
       Maybe.Nope,
-      Maybe.Nope)
+      Maybe.Nope,
+      isHidden)
```

Comment:
This should always be false here. I.e., a ChoiceBranchImpliedSequence cannot be created for a group ref. So you can write
```
{
Assert.invariant(!isHidden)
false
}
```

Reply:
The Runtime structures no longer have an isHidden member